docs/docs/providers/reddit.md

```diff
@@ -47,25 +47,28 @@ This Provider template only has a one hour access token to it and only has the "
 ```js
 providers: [
   {
-    id: "reddit",
-    name: "Reddit",
-    clientId: process.env.REDDIT_CLIENT_ID,
-    clientSecret: process.env.REDDIT_CLIENT_SECRET,
-    scope: "identity mysubreddits read", //Check Reddit API Documentation for more. The identity scope is required.
-    type: "oauth",
-    version: "2.0",
-    params: { grant_type: "authorization_code" },
-    accessTokenUrl: " https://www.reddit.com/api/v1/access_token",
-    authorizationUrl:
-      "https://www.reddit.com/api/v1/authorize?response_type=code&duration=permanent",
-    profileUrl: "https://oauth.reddit.com/api/v1/me",
-    profile: (profile) => {
-      return {
-        id: profile.id,
-        name: profile.name,
-        email: null,
-      }
-    },
+    id: 'reddit',
```

Comment:
Let's use `RedditProvider`, so you only need to override what's different from the default config.

Reply:
sounds good 👍